subprojects/docs/src/docs/release/notes.md

```diff
@@ -166,24 +175,23 @@ The `--tests` filters are now always applied on top of the filtering defined in
 
 The base package has been renamed from `org.gradle.script.lang.kotlin` to `org.gradle.kotlin.dsl` so any explicit imports need to be updated after upgrading.
 
-
 ## External contributions
 
 We would like to thank the following community members for making contributions to this release of Gradle.
 
- - [Jörn Huxhorn](https://github.com/huxi) - Replace uses of `Stack` with `ArrayDeque` (#771)
- - [Björn Kautler](https://github.com/Vampire) - Fix WTP component version (#2076)
- - [Bo Zhang](https://github.com/blindpirate) - Add support for 'console' output type of CodeNarc plugin (#2170)
- - [Bo Zhang](https://github.com/blindpirate) - Fix infinite loop when using `Path` for task property (#1973)
- - [Bo Zhang](https://github.com/blindpirate) - Contributions to consistent --tests option handling (#2172)
- - [Marcin Zajączkowski](https://github.com/szpak) - Add `@since` tag to `Project.findProperty()` (#2403)
- - [Seth Jackson](https://github.com/sethjackson) - Fix the default daemon JVM args on Java 8 (#2310)
- - [Ismael Juma](https://github.com/ijuma) - Update default Zinc compiler version to 0.3.15 with preliminary Java 9 support (#2420)
- - [Krzysztof Ropiak](https://github.com/krro) - Sets annotation processor classpath in Scala compilation task (#2281)
- - [Dave Brewster](https://github.com/dbrewster) - Add support for caching Scala compilation (#1958)
- - [Steve Dougherty](https://github.com/Thynix) - Prevent spurious gradle-wrapper.jar changes (#2183)
- - [Yannick Welsch](https://github.com/ywelsch) - Use GNU-style release flag for Java 9 compiler (#2474)
- - [Mike Kobit](https://github.com/mkobit) - Add Worker API classes to JavaDocs (#2372)
+ - [Dave Brewster](https://github.com/dbrewster) - Add support for caching Scala compilation (gradle/gradle#1958)
```

Comment:
Should we sort the list of contributors in any way? I remember sorting them alphabetically for some earlier release. Now the order just seems to be random.